tests/console/sysctl.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test sysctl because it can go wrong https://bugzilla.opensuse.org/show_bug.cgi?id=1077746
+# Maintainer: Bernhard M. Wiedemann <bwiedemann suse.de>
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run {
+    select_console 'user-console';
+    validate_script_output 'sysctl -w vm.swappiness=59 ; cat /proc/sys/vm/swappiness', sub {
```

Comment:
Is the user really allowed to call sysctl? I would also recommend to call the sysctl command explicitly with assert_script_run. It shouldn't fall, right?

Reply:
changed it to use root-console as otherwise proc files are not writable